pkgs/tools/misc/engage/default.nix

```diff
@@ -1,4 +1,5 @@
 { lib
+, pkgs
```

Comment:
Is this necessary for `installShellFiles`?

It shoulf be fine adding it as an input here.

Reply:
```diff
--- a/pkgs/tools/misc/engage/default.nix
+++ b/pkgs/tools/misc/engage/default.nix
@@ -1,5 +1,4 @@
 { lib
-, pkgs
 , rustPlatform
 , fetchgit
 }:
@@ -20,7 +19,7 @@ rustPlatform.buildRustPackage {

   cargoHash = "sha256-+4uqC0VoBSmkS9hYC1lzWeJmK873slZT04TljHPE+Eo=";

-  nativeBuildInputs = with pkgs; [
+  nativeBuildInputs = [
     installShellFiles
   ];
```
gives me
```
$ nix shell .#engage
warning: Git tree '/home/charles/science/nix/nixpkgs' is dirty
error:
       … while evaluating a branch condition

         at /nix/store/mx1wcv20iqwcfchb8p6d4hhnhrg9jqhj-source/lib/customisation.nix:173:8:

          172|
          173|     in if missingArgs == [] then makeOverridable f allArgs else abort error;
             |        ^
          174|

       … while calling the 'attrNames' builtin

         at /nix/store/mx1wcv20iqwcfchb8p6d4hhnhrg9jqhj-source/lib/customisation.nix:133:21:

          132|       # wouldn't be passed to it
          133|       missingArgs = lib.attrNames
             |                     ^
          134|         # Filter out arguments that have a default value

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: undefined variable 'installShellFiles'

       at /nix/store/mx1wcv20iqwcfchb8p6d4hhnhrg9jqhj-source/pkgs/tools/misc/engage/default.nix:23:5:

           22|   nativeBuildInputs = [
           23|     installShellFiles
             |     ^
           24|   ];
```

Reply:
Oh, because it's supposed to be

```diff
--- a/pkgs/tools/misc/engage/default.nix
+++ b/pkgs/tools/misc/engage/default.nix
@@ -1,5 +1,5 @@
 { lib
-, pkgs
+, installShellFiles
 , rustPlatform
 , fetchgit
 }:
@@ -20,7 +20,7 @@ rustPlatform.buildRustPackage {

   cargoHash = "sha256-+4uqC0VoBSmkS9hYC1lzWeJmK873slZT04TljHPE+Eo=";

-  nativeBuildInputs = with pkgs; [
+  nativeBuildInputs = [
     installShellFiles
   ];
```